SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -7,6 +7,190 @@
 
 #include "SpatialGDKEditorSettings.generated.h"
 
+USTRUCT()
+struct FWorldLaunchSection
+{
+	GENERATED_BODY()
+
+	FWorldLaunchSection()
+		: Dimensions(2000, 2000)
+		, ChunkEdgeLengthMeters(50)
+		, StreamingQueryInterval(4)
+		, SnapshotWritePeriodSeconds(0)
+	{
+		LegacyFlags.Add(TEXT("bridge_qos_max_timeout"), TEXT("0"));
+		LegacyFlags.Add(TEXT("bridge_soft_handover_enabled"), TEXT("false"));
+		LegacyFlags.Add(TEXT("enable_chunk_interest"), TEXT("false"));
+	}
+
+	/** The size of the grid squares that the world is divided into, in “world units” (an arbitrary unit that workers can interpret as they choose. */
```

Comment:
A bunch of these comments need updating

Reply:
@padhansell I more or less just copied them from the docs we have on launch configs, could you help me identify which ones that you think are wrong and how they need updating so that I can follow up with @NotRobSheridan about them :)

Reply:
Ah I getcha. It looks like some of them have the wrong comment for the property. Eg:

`	/** The frequency in seconds to write snapshots of the simulated world. */
	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "Chunk edge length in meters"))
	int32 ChunkEdgeLengthMeters;`

Reply:
Yes classic copy paste error! 🤦‍♂️  thanks for pointing it out :)